src/utils/JsonSchemaUtils.js

```diff
@@ -1,4 +1,4 @@
-import Ajv2020 from 'ajv/dist/2020';
+import Ajv2019 from 'ajv/dist/2019';
```

Comment:
```suggestion
import Ajv2019 as Ajv from 'ajv/dist/2019';
```

Could we do this instead to make it easy to switch out?

Reply:
oh true - I did `Ajv = Ajv2019` down below but this is probably easier